buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -2522,6 +2526,9 @@ class BeamModulePlugin implements Plugin<Project> {
           project.exec {
             environment "EXPANSION_JAR", expansionJar
             environment "EXPANSION_PORT", javaExpansionPort
+            if (config.additionalArgs.containsKey("KAFKA_BOOTSTRAP_SERVER")) {
+              environment "BOOTSTRAP_SERVER", config.additionalArgs.get("KAFKA_BOOTSTRAP_SERVER")
```

Comment:
Consider use the same naming (`BOOTSTRAP_SERVER` -> `KAFKA_BOOTSTRAP_SERVER`)?

And could`config.additionalArgs` be generalized? e.g., naming this variable as config.additionalEnvs, and for all keys, set the corresponding environment variable accordingly;

or actually these environment variables are effective once set so its can be set in gradle file directly and does not need to touch BeamModulePlugin? Or there could be a better way.

In any case the purpose is that keep BeamModulePlugin logic generic so Kafka things/namings does not explicitly appear here

Reply:
thats a good improvement, I'll do that